sonic-chassisd/scripts/chassisd

```diff
@@ -296,37 +305,34 @@ class ModuleUpdater(logger.Logger):
                 prev_status = self.get_module_current_status(key)
                 self.module_table.set(key, fvs)
 
-                # Construct key for down_modules dict. Example down_modules key format: LINE-CARD0|<hostname>
-                fvs = self.hostname_table.get(key)
-                if isinstance(fvs, list) and fvs[0] is True:
-                    fvs = dict(fvs[-1])
-                    hostname = fvs[CHASSIS_MODULE_INFO_HOSTNAME_FIELD]
-                    down_module_key = key+'|'+hostname
-                else:
-                    down_module_key = key+'|'
-
-                if module_info_dict[CHASSIS_MODULE_INFO_OPERSTATUS_FIELD] != str(ModuleBase.MODULE_STATUS_ONLINE):
-                    if prev_status == ModuleBase.MODULE_STATUS_ONLINE:
```

Comment:
Why was prev_status removed? This is legacy code that is not executed for the smart switch.

Reply:
This not removed, but the indentation has changed as we are checking for is_smartswitch().

Reply:
Even I think prev_status seems to be removed and not used.